llama-index-integrations/vector_stores/llama-index-vector-stores-relyt/pyproject.toml

```diff
@@ -0,0 +1,66 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.llamahub]
+contains_example = false
+import_path = "llama_index.vector_stores.relyt"
+
+[tool.llamahub.class_authors]
+RelytVectorStore = "llama-index"
+
+[tool.mypy]
+disallow_untyped_defs = true
+exclude = ["_static", "build", "examples", "notebooks", "venv"]
+ignore_missing_imports = true
+python_version = "3.8"
+
+[tool.poetry]
+authors = ["Your Name <you@example.com>"]
+description = "llama-index vector_stores relyt integration"
+exclude = ["**/BUILD"]
+license = "MIT"
+name = "llama-index-vector-stores-relyt"
+readme = "README.md"
+version = "0.1.0"
+
+[tool.poetry.dependencies]
+python = ">=3.8.1,<4.0"
+llama-index-core = "^0.10.1"
```

Comment:
We should probably just make all dependencies here, rather than as extras? Seems like pgvecto-rs is needed to even import this package